webapp-client/src/component/ContentPane.js

```diff
@@ -0,0 +1,67 @@
+import Category from './Category';
+import Item from './Item';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class ContentPane extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      displayCategory: <Category categoryName="All" />,
+      items: [],
+    };
+    this.setContentPane = this.setContentPane.bind(this);
+  }
+
+  async componentDidMount() {
+    const items = await this.getItems(this.props);
+    this.setState({ items });
+  }
+
+  async getItems(props) {
+    const { currentCategory, defaultCategory, userId } = props;
+    let url, response, items;
+    if (currentCategory === defaultCategory) {
+      url = process.env.REACT_APP_API_URL + '/item/getItems/' + userId;
+      response = await fetch(url);
+      items = await response.json();
+    } else {
+      url =
+        process.env.REACT_APP_API_URL +
+        '/category/getCategoryItems/' +
+        currentCategory;
```

Comment:
Ik this is done by formatter, do you think it's easier to read if we have url on one line

Reply:
I think it's fine to have it on multiple lines. No more than 80 characters on a single line is pretty standard because it can roll off the screen in some editors (to my understanding).